web/packages/teleterm/src/env.ts

```diff
@@ -0,0 +1,24 @@
+interface Env {
+  prehogAddress: string;
+  feedbackAddress: string;
+}
```

Comment:
Please add a comment explaining what it is and how it differs from RuntimeSettings.

Reply:
Done